components/layout_2020/style_ext.rs

```diff
@@ -165,6 +173,82 @@ impl ComputedValuesExt for ComputedValues {
             self.writing_mode,
         )
     }
+
+    /// Returns true if this style has a transform, or perspective property set.
+    fn has_transform_or_perspective(&self) -> bool {
+        !self.get_box().transform.0.is_empty() || self.get_box().perspective != Perspective::None
+    }
+
+    /// Get the effective z-index of this fragment. Z-indices only apply to positioned element
```

Comment:
uber-nit: should be "positioned elements"

Reply:
Fixed this as well. Thanks for pointing it out.